solutions/swb-reference/integration-tests/tests/multiStep/costCenter.test.ts

```diff
@@ -48,6 +48,7 @@ describe('multiStep costCenter test', () => {
     expect(updatedCostCenterB).toMatchObject({ name, description });
 
     console.log('Delete Cost Center B');
-    await expect(adminSession.resources.costCenters.costCenter(createdCostCenterB.id).delete()).resolves;
+    // eslint-disable-next-line no-unused-expressions
+    expect(await adminSession.resources.costCenters.costCenter(createdCostCenterB.id).delete()).resolves;
```

Comment:
Are we sure the `await` is supposed to be in the expect?  The example shown [here](https://jestjs.io/docs/asynchronous) shows the format of the original code. 

```
test('the data is peanut butter', async () => {
  await expect(fetchData()).resolves.toBe('peanut butter');
});
```

Reply:
When I ran it the example way, I got one of those "a process is still open you're test might not have finished" jest errors sometimes. I am sure the await outside works while the await inside might cause async issues. I think it might be because the check is just that is resolves, not the resolution value like you have in the example maybe?

Reply:
I recommend

`await expect(adminSession.resources.costCenters.costCenter(createdCostCenterB.id).delete()).resolves.toBeUndefined()`

Reply:
We'll look to do that in the future!